### DISCUSSION TITLE
RA loading fine then get aborated (pi zero w), any suggestion please?

### DISCUSSION THREAD

**Discussion by funky1983** [mKeRix/room-assistant#1605] [Q&A]
thats the log, dont understand the error:

12/07/2024, 17:41:03 - info - IntegrationsModule: Loading integrations: none
12/07/2024, 17:41:06 - info - NestFactory: Starting Nest application...
12/07/2024, 17:41:07 - info - InstanceLoader: AppModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: ConfigModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: NestEmitterModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: IntegrationsModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: DiscoveryModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: PrometheusModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: EventsModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: TerminusModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: ClusterModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: ScheduleModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: EntitiesModule dependencies initialized
12/07/2024, 17:41:07 - info - InstanceLoader: StatusModule dependencies initialized
12/07/2024, 17:41:15 - info - WebSocketsController: EventsGateway subscribed to the "subscribeEvents" message
12/07/2024, 17:41:15 - info - RoutesResolver: EntitiesController {/entities}:
12/07/2024, 17:41:15 - info - RouterExplorer: Mapped {/entities, GET} route
12/07/2024, 17:41:15 - info - RoutesResolver: StatusController {/status}:
12/07/2024, 17:41:15 - info - RouterExplorer: Mapped {/status, GET} route
12/07/2024, 17:41:15 - info - RoutesResolver: PrometheusController {/metrics}:
12/07/2024, 17:41:15 - info - RouterExplorer: Mapped {/metrics, GET} route
12/07/2024, 17:41:15 - info - ConfigService: Loading configuration from /opt/nodejs/lib/node_modules/room-assistant/dist/config/definitions/default.js, /opt/nodejs/lib/node_modules/room-assistant/dist/config/definitions/custom-environment-variables.yml (Current: /home/amir)
12/07/2024, 17:41:16 - info - ClusterService: Starting mDNS advertisements and discovery
12/07/2024, 17:41:16 - info - NestApplication: Nest application successfully started
12/07/2024, 17:41:17 - info - ClusterService: pizerolivingroom has been elected as leader
12/07/2024, 17:41:17 - info - ClusterService: Added 192.168.0.103:6425 to the cluster with id main-lab

  #  node[1806]: void node::cares_wrap::{anonymous}::GetAddrInfo(const v8::FunctionCallbackInfo<v8::Value>&) at ../src/cares_wrap.cc:1575
  #  Assertion failed: args[4]->IsUint32()

----- Native stack trace -----

----- JavaScript stack trace -----

1: getaddrinfo_0_11 (/opt/nodejs/lib/node_modules/room-assistant/node_modules/mdns/lib/resolver_sequence_tasks.js:117:21)
2: /opt/nodejs/lib/node_modules/room-assistant/node_modules/mdns/lib/resolver_sequence_tasks.js:153:7
3: getaddrinfo (/opt/nodejs/lib/node_modules/room-assistant/node_modules/mdns/lib/resolver_sequence_tasks.js:152:14)
4: next (/opt/nodejs/lib/node_modules/room-assistant/node_modules/mdns/lib/browser.js:109:21)
5: on_resolver_done (/opt/nodejs/lib/node_modules/room-assistant/node_modules/mdns/lib/resolver_sequence_tasks.js:33:11)
6: MDNSService.self.watcher.callback (/opt/nodejs/lib/node_modules/room-assistant/node_modules/mdns/lib/mdns_service.js:18:40)

Aborted

**Comment by jpparamor** [comment]
I'm also receiving exactly the same error on a new install of room assistant on 2 Raspberry Pi Zero W's.

**Reply by funky1983** [reply]
downgrade the nodejs to 16

**Reply by MarcovDam** [reply]
How?
